cmd/mysql-operator/app/mysql_operator.go

```diff
@@ -45,15 +45,15 @@ const (
 
 // resyncPeriod computes the time interval a shared informer waits before
 // resyncing with the api server.
-func resyncPeriod(s *options.MySQLOperatorServer) func() time.Duration {
+func resyncPeriod(s *opratoropts.MySQLOperatorOpts) func() time.Duration {
 	return func() time.Duration {
 		factor := rand.Float64() + 1
 		return time.Duration(float64(s.MinResyncPeriod.Nanoseconds()) * factor)
 	}
 }
 
 // Run starts the mysql-operator controllers. This should never exit.
-func Run(s *options.MySQLOperatorServer) error {
+func Run(s *opratoropts.MySQLOperatorOpts) error {
```

Comment:
Minor typo here (opratoropts => operatoropts)

Reply:
Thanks for the review.